blog_article/blog_article.adoc

```diff
@@ -0,0 +1,161 @@
+interoperability between all messaging middleware". Using the RabbitMQ broker as an example, we want to take
+a closer look at the extent to which AMQP is supported and how different versions interoperate with each other.
+Therefore, in the following, we will compare the use of three Java client libraries: RabbitMQ, SwiftMQ and Qpid JMS.
+The RabbitMQ broker requires an activated AMQP 1.0 plugin for this. +
+The competing consumer and publish-subscribe patterns were implemented using these clients within this
+https://github.com/NovatecConsulting/showcase-rabbitmq/tree/main[repository]. Feel free to have a look at the implementations
+and how they differ from each other!
+
+== RabbitMQ Support for AMQP 1.0
+Most of the established brokers only or mainly support AMQP 1.0 since this protocol version was introduced as an OASIS
+standard. In contrast to that, RabbitMQ focuses on AMQP 0.9.1, and AMQP 1.0 support can only be added via a plugin.
+Nevertheless, since SwiftMQ, ActiveMQ and Qpid are JMS-based, there are fewer AMQP 1.0 client libraries available for
+programming languages other than Java. +
+In case a broker is needed which supports AMQP 0.9.1 and AMQP 1.0 on equal levels, the choice will remain limited:
+
+|===
+|Broker Implementation |AMQP Protocol Support
+
+|https://www.rabbitmq.com/[RabbitMQ]
+|mainly supports AMQP 0.9.1; AMQP 1.0 support by plugin
+
+|https://www.swiftmq.com/[SwiftMQ]
+|mainly supports AMQP 1.0; limited AMQP 0.9.1 support
+
+|https://activemq.apache.org/components/artemis/[ActiveMQ Artemis]
+|supports AMQP 1.0
+
+|https://activemq.apache.org/components/classic/[ActiveMQ "Classic"]
+|supports AMQP 1.0 via a transport connector
+
+|https://qpid.apache.org/components/broker-j/index.html[Qpid Broker-J]
+|supports AMQP 0.8 to 1.0
+
+|===
+
+Coming back to RabbitMQ as an example, their Java client only supports sending and receiving AMQP 0.9.1 messages.
+So another library is needed to use the AMQP 1.0 plugin. At the moment of writing this article and as also stated
+in the documentation of the plugin, there are only a few clients for AMQP 1.0, especially for Java:
+
+* https://www.swiftmq.com/docs/docs/client/amqp/[SwiftMQ]:
+The plugin's developers tested the functionalities mainly using SwiftMQ.
+* https://qpid.apache.org/releases/qpid-proton-j-0.33.8/[Qpid Proton-J]:
+The documentation is not that detailed and does not provide examples.
+* https://qpid.apache.org/components/jms/index.html[Qpid JMS]:
+Uses Proton-J internally.
+
+As already mentioned, AMQP 1.0 is not based on queues or topics. The target (or source) of a message has to be
+specified using the address field of an AMQP 1.0 message like it is described in the https://github.com/rabbitmq/rabbitmq-server/tree/master/deps/rabbitmq_amqp1_0#routing-and-addressing[plugin documentation].
+
+Since this concept does not exist in AMQP 1.0, the Java clients do not offer the possibility to manage queues,
```

Comment:
"Since this concept does not exist in AMQP 1.0" does not fit here I think. In the last section you wrote that AMQP 1.0 is using the address field for specify the target